src/main/menu.ts

```diff
@@ -169,6 +169,13 @@ function getMenuTemplate(logHandler: LogHandler): rawMenuItem[] {
                 translate: 'global_menu_file_quit_desktop',
                 role: 'quit',
               },
+              {
+                translate: 'global_menu_file_reduce_desktop',
+                accelerator: 'Escape',
```

Comment:
I wouldn't introduce the escape shortcut here, escape also does other things and I think the normal user gets confused when escape results in the window being invisible.